docs/user/simultaneous_use.rst

```diff
@@ -0,0 +1,33 @@
+Simultaneous session limits (Simultaneous-Use)
+==============================================
+
+``Simultaneous-Use`` limits how many sessions a user can have active at
+the same time.
+
+During authorization, OpenWISP RADIUS counts the user's active sessions
+and compares them with the limit set in their RADIUS group. If the number
+of active sessions is **equal to or greater than** the limit, access is
+denied with:
```

Comment:
Let's simplify this text.. it's quite convoluted.. "if simultaneous-use is set, the user can have at max that amount of sessions open", something along these lines.